google-beta/node_config.go

```diff
@@ -16,6 +20,23 @@ var defaultOauthScopes = []string{
 	"https://www.googleapis.com/auth/trace.append",
 }
 
+// This validates for a tuple-3 of positive integers. Spaces and tabs are supported as separators
+var tcpMemoryRegexp = regexp.MustCompile(`^([1-9][0-9]+|0)[ \t]+([1-9][0-9]+|0)[ \t]+([1-9][0-9]+|0)$`)
+
+// Keep track of the original sysctl keys since we replace dots by underscores to make the keys more TF idiomatic.
+// Ref: https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/NodeConfig#LinuxNodeConfig
+var sysAttrsMap = map[string]string{
```

Comment:
Is this roughly a fixed list, or is it one we expect to grow over time? Ideally, entries are added on roughly a yearly cadence.

Reply:
I expect this to grow overtime and quite honestly have mixed feelings about this conversion... Agreed with your point below, we could relax on the validations and cross fingers that the GCP APIs return good error messages when given bad input.